arviz/plots/posteriorplot.py

```diff
@@ -184,7 +184,7 @@ def plot_posterior(
 
         >>> az.plot_posterior(data, var_names=['mu'], kind='hist')
 
-    Change size of HPD interval
+    Change size of highest density interval
```

Comment:
Thanks for catching my misses

Reply:
I did not remove any function (at least not on purpose, haha) I just marked `plot_hpd` as deprecated in favor of `plot_hdi`. And I replaced the strings `hpd` to `hdi`.

Reply:
Ah yes you're right I see it now